supervisor/config/config.go

```diff
@@ -53,7 +53,7 @@ func (c Config) Write() error {
 	if err != nil {
 		return fmt.Errorf("could not encode configuration: %w", err)
 	}
-	if err := ioutil.WriteFile(confPath, []byte(out.String()), 0700); err != nil {
+	if err := ioutil.WriteFile(confPath, []byte(out.String()), 0644); err != nil {
```

Comment:
This one will likely panic on windows because modes other than 0600 and 0400 are not supported there.

Reply:
[Doesn't seem like Windows version is implemented](https://github.com/mysteriumnetwork/node/blob/master/supervisor/config/const_windows.go), also the change is from 0700 which would also panic?

Reply:
No, 0700 should be fine. Windows version of supervisor is actively used.

Reply:
> No, 0700 should be fine. Windows version of supervisor is actively used.

I've [split the config](https://github.com/mysteriumnetwork/node/pull/4992/commits/716f1dd5ec692ea5d106d32a6e23aec1510546c9) files leaving 0700 on Windows and 0644 on Mac & Linux